### PR TITLE
Browser cleanup

### PR DESCRIPTION
**Pull request by quardbreak**

### CODE REVIEW

File: code/datums/browser.dm [hunk 1]
```diff
@@ -109,94 +119,47 @@
 	if(written_text)
 		. = user.handle_reading_literacy(user, .)
 
-/datum/browser/proc/open(var/use_onclose = 1)
```
Comment (afterthought2) [inline code/datums/browser.dm:112]: You've searched for all instances of `open` and `update` and made sure they aren't using the old named args? People love using named args for these kinds of things.

File: code/datums/browser.dm [hunk 2]
```diff
@@ -109,94 +119,47 @@
...
-// Optionally, specify the "ref" parameter as the controlled atom (usually src)
-// to pass a "close=1" parameter to the atom's Topic() proc for special handling.
-// Otherwise, the user mob's machine var will be reset directly.
-//
-/proc/onclose(mob/user, windowid, var/atom/ref=null)
-	if(!user || !user.client) return
-	var/param = "null"
-	if(ref)
-		param = "\ref[ref]"
+/// Register windowclose to be called, optionally with ref, when the window window_id is closed
```
Comment (afterthought2) [inline code/datums/browser.dm]: Why are we removing the documentation here? This is a super non-intuitive feature, I'd rather all this stuff stayed.

File: code/game/objects/items/weapons/cards_ids.dm [hunk 3]
```diff
@@ -222,7 +222,6 @@ var/global/const/NO_EMAG_ACT = -50
 		send_rsc(user, side, "side.png")
 	var/datum/browser/written/popup = new(user, "idcard", name, 600, 250)
 	popup.set_content(dat())
-	popup.set_title_image(usr.browse_rsc_icon(src.icon, src.icon_state))
```
Comment (afterthought2) [inline code/game/objects/items/weapons/cards_ids.dm:225]: Why are these being removed?

Comment (quardbreak) [reply code/game/objects/items/weapons/cards_ids.dm:225]: Legacy code which doesn't even work.

Comment (afterthought2) [reply code/game/objects/items/weapons/cards_ids.dm:225]: Fair enough, I guess. You should probably remove `proc/set_title_image` in that case.

**Review by afterthought2 — APPROVED** [review]

**Pull request merged**